src/xenia/app/emulator_window.cc

```diff
@@ -174,10 +176,37 @@ bool EmulatorWindow::Initialize() {
   // FIXME: This code is really messy.
   auto main_menu = MenuItem::Create(MenuItem::Type::kNormal);
   auto file_menu = MenuItem::Create(MenuItem::Type::kPopup, L"&File");
+  // Recent Sub tab
+  auto recent_list = MenuItem::Create(MenuItem::Type::kPopup, L"&Open Recent");
```

Comment:
Minor point, but here the keyboard alt-combo is assigned to alt+o which duplicates open - considering moving this to alt+r (`Open &Recent`)